### ISSUE TITLE
Terminal doesn't work in Ubuntu Unity 24.10

### ISSUE THREAD

**Issue by fmixolydian** [SpartanJ/ecode#366]
Hello, I recently installed `ecode` on an Ubuntu Unity 24.10 installation through the AppImage; i copied the AppImage to the `/usr/local/bin` directory and installed `libfuse2t64`.

Here's a screenshot:
![image](https://github.com/user-attachments/assets/f2dadca7-3fb8-460b-88d8-c61b992059c4)

**Comment by SpartanJ** [comment]
Mmhh that's odd, maybe you can try to configure a different shell in `Settings -> Terminal -> Configure Terminal Shell`. It looks like the shell isn't responding. I don't have Ubuntu 24.10 installed, I'll check just in case (in 24.04 works just fine).

**Comment by fmixolydian** [comment]
Yeah, I did that. I chose `/usr/bin/bash`. Also, I'm in the process of installing the `tar.gz` version. I hope that'll work better.

**Comment by fmixolydian** [comment]
Nope... the `tar.gz` version is still broken.
![Screenshot from 2024-12-11 22-57-06](https://github.com/user-attachments/assets/29b00543-7dac-461c-ae34-83d4afcc4bee)

**Comment by fmixolydian** [comment]
Also, it used to work in my previous Debian 12 installation. I'm using the same computer, but with a different hard drive. Just in case, here's the output of  `uname -a`:
```
Linux <...> 6.11.0-9-generic #9-Ubuntu SMP PREEMPT_DYNAMIC Mon Oct 14 13:19:59 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by SpartanJ** [comment]
I can reproduce the issue on Ubuntu 24.10. So I'll investigate. I've no idea what's going on, also the window manager change the colors of the app to match the default theme WTF.

**Comment by SpartanJ** [comment]
Oh, it's crazy, if you wait long enough (a few minutes) the shell will end up loading, it also the forked process will use 100% of the CPU while "loads". There's something really wrong.

**Comment by SpartanJ** [comment]
Found the issue, will push a fix later.

**Comment by fmixolydian** [comment]
> Oh, it's crazy, if you wait long enough (a few minutes) the shell will end up loading, it also the forked process will use 100% of the CPU while "loads". There's something really wrong.

I can confirm this is the truth. 
![Screenshot from 2024-12-12 00-57-29](https://github.com/user-attachments/assets/4fa5ebbb-664c-4342-ae14-f10f67b91bed)

For several minutes, the fans are pretty loud (they're usually silent), and one core is always at 100%.

However, even after waiting several minutes, nothing seems to load.

**Comment by SpartanJ** [comment]
I just pushed a fix. You'll be able to get the [nightly build](https://github.com/SpartanJ/eepp/releases/tag/nightly) in about 15 minutes from this message.

**Comment by fmixolydian** [comment]
Nice. I can confirm that it works now. Thanks.

![Screenshot from 2024-12-12 15-14-10](https://github.com/user-attachments/assets/37cb33c0-913a-4fd6-bcb7-7e2ffe8787db)
